documentation/library_guide/oneDPL_2022.0_changes.rst

```diff
@@ -0,0 +1,40 @@
+oneDPL 2022.0 Changes
+#####################
+
+This page lists breaking changes in oneDPL 2022.0 from oneDPL 2021.
+
+Support of C++11 and C++14 has been discontinued. 
+
+The internal parameter n of discard_block_engine was changed
+ from int to std::size_t type to align implementation with
+ С++ std proposal https://cplusplus.github.io/LWG/issue3561.
+ This change lets users to utilize the full range of values for P and R
+ template parameters of discard_block_engine.
+ 
+ .. Note::
+
+ Note: it may require rebuilding users’ code.
```

Comment:
Note: You may need to rebuild your code.

Reply:
Applied the suggestion.